src/screens/application/Preview.tsx

```
@@ -141,19 +141,28 @@ const Preview: React.FC = () => {
 
 								{secondItem && (
 									<Box flex={1} alignItems="center">
 										<Text {...labelStyles}>
 											{secondItem.label}
 										</Text>
-										<Text {...valueStyles}>
-											{secondItem.value}
-										</Text>
+										{secondItem.label === 'Student Type' ||
+										secondItem.value === '' ? (
+											<Text {...valueStyles}>-</Text>
```

ESLint

[error] Expected indentation of 12 tabs but found 11.

(indent)